RevenueCatUI/CustomerCenter/Views/FeedbackSurveyView.swift

```diff
@@ -25,20 +25,37 @@ import SwiftUI
 @available(visionOS, unavailable)
 struct FeedbackSurveyView: View {
 
-    @ObservedObject
-    var feedbackSurveyData: FeedbackSurveyData
+    @StateObject
+    private var viewModel: FeedbackSurveyViewModel
+
+    @Environment(\.localization)
+    private var localization: CustomerCenterConfigData.Localization?
```

Comment:
Sorry for the back and forth but I'm wondering if we should make this not nullable after all... When these views are created, the configuration should have always loaded so it would be safe to assume this not to be null. That way we don't need checks all over the place for nullability. Or are there any views that require localization before the configuration has loaded?

Reply:
the configuration is the first thing that gets loaded so there should be localization in all screens